install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -120,5 +120,6 @@ spec:
             requests:
               cpu: 200m
               memory: 384Mi
+              ephemeral-storage: 1Gi
```

Comment:
This file should not change outside of releases. If you want to change something, you need to:
* Do it in the `packaging/...` doirectory
* Change it in the Helm Chart and call `make helm_install` to generate the YAML files

Reply:
Removed. This was added by mistake.